core/src/main/scala/cats/data/ContT.scala

```diff
@@ -45,6 +45,8 @@ sealed abstract class ContT[M[_], A, +B] extends Serializable {
       M.defer(run(fn3))
     }
   }
+
+  final def eval(implicit M: Monad[M], ev: B <:< A): M[A] = run(b => M.pure(ev(b)))
```

Comment:
Why monad here? Seems you are only using pure?

Reply:
Thanks. Yeah, cargo-culted from Haskell wtithout thinking too much about it. Will relax to `Applicative`

Reply:
Actually, that is required - it needs `Applicative` for `eval` and `FlatMap` for `liftF`